### PR TITLE
Hold ethereum-utils version

### PR DESCRIPTION
**Pull request by Elfoniok**
Fixes this issue:
https://github.com/golemfactory/golem/issues/1755

Signed-off-by: Lukasz Foniok <lukaszfoniok@gmail.com>

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@ cffi==1.10.0
 web3==3.8.0
 ethereum==1.6.1
 ethereum-abi-utils==0.4.0
+ethereum-utils==0.5.1
```
Comment (maaktweluit) [inline requirements.txt]: Would be nice to add a comment why its fixed
Maybe you can link the issue to upgrade the package, is that made?

**Review by maaktweluit — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (Elfoniok) [reply requirements.txt]: Comment in requirement file?

Comment (etam) [reply requirements.txt]: Like in requirements-test.txt. Start with `#`.

Comment (Elfoniok) [reply requirements.txt]: That is a new custom but ok

**Review by maaktweluit — APPROVED** [review]

**Comment by maaktweluit** [comment]
Thanks for the update!

Last weeks we had some breaking frozen packages and found out future self will be happy with these comments. This way we can upgrade more safely and quickly if we know the root cause.

File: requirements.txt [hunk 2]
```diff
@@ -20,6 +20,8 @@ cffi==1.10.0
 web3==3.8.0
 ethereum==1.6.1
 ethereum-abi-utils==0.4.0
+#Package changed name
```
Comment (etam) [inline requirements.txt]: That doesn't explain anything.
That would be better:
`Newer version contains no code. Only deprecation warning.`

**Review by etam — CHANGES_REQUESTED** [review]

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**